### ISSUE TITLE
NTR Fibro-osseous lesion

### ISSUE THREAD

**Issue by jmfostel** [mgijax/mammalian-phenotype-ontology#4155]
this term is found in MPATH and we were hoping uPHENO could import it.
MPATH:590

i have not been able to find a reasonable replacement.  the synonyms in MPATH are not exact.
our pathology group suggested osteofibrosis as a synonym

fibrous dysplasia MONDO:0000845 is a genetic disorder; we need it to be seen after chemical exposure.
osteodystrophy, non-human animal MONDO:1011951 does not come with a definition; our review of google suggests Osteodystrophy is a general term for defective or abnormal bone development, often linked to chronic kidney disease.
osteodysplasia is not in MONDO but related terms are genetic disorders.
osteofibrosis is not in OLS other than the MPATH term

we define fibro-osseous lesion this way:
NIEHS pathologist, verbal discussion:  characterized by proliferation of stroma, i.e. supportive tissue consisting of connective tissue and blood vessels, with proliferation of osteoblasts and osteoclasts cell populations.  

from our Non Neoplastic Lesion Atlas:
https://ntp.niehs.nih.gov/atlas/nnl/musculoskeletal-system/bone/Fibro-osseousLesion

These lesions are characterized by partial or complete replacement of bony trabeculae and marrow cavity by fibrovascular stroma containing fibroblasts, osteoclasts, and osteoblasts embedded in eosinophilic matrix and. The histopathologic features are similar to fibrous osteodystrophy; however, FOLs occur in the absence of parathyroid or renal alterations, typically arise as focal lesions within the metaphyseal or endocortical regions, and may progress to involve larger areas of the bone.

from this paper:  PMID: 38705816
craniofacial fibro-osseous lesions represent a diverse spectrum of pathologic conditions where fibrous tissue replaces healthy bone, resulting in the formation of irregular, woven bone. 

from MPATH:
The benign fibro-osseous lesions (BFOL) represent a clinically diverse group of disorders of bone that share similar histopathologic features. As a group, they are relatively common in the craniofacial complex, especially the jaws. Common in some inbred strains notably B6C3F1 females. Note that many of the synonyms have been used historically and may not be accurate. See: Albassam MA, Courtney CL (1996) Nonneoplastic and neoplastic lesions of the bone. In: Mohr U, Dungworth DL, Capen CC, Carlton WW, Sundberg JP, Ward JM (eds) Pathobiology of the aging mouse. Vol 2. Musculoskeletal system. ILSI Press, Washington D.C., pp 425-437. AND Albassam MA, Wojcinski ZW, Barsoum NJ, Smith, GS (1991) Spontaneous fibro-osseous proliferative lesions in the sternums and femurs of B6C3F1 mice. Vet Path 28: 381-388.

thanks for your help!

**Comment by anna-anagnostop** [comment]
Would the existing term bone fibrous dysplasia [MP:0031348](https://www.informatics.jax.org/vocab/mp_ontology/MP:0031348) work for you? Let us know if we should add fibro-osseous lesion as a synonym to MP:0031348 or should we change the primary label to fibro-osseous lesion and add fibrous dysplasia as a narrow synonym?
Do we need to tweak our definition for MP:0031348?

**Comment by jmfostel** [comment]
thank you!  i think that this is a good fit.  can you add fibro-osseous lesion as a synonym?
